workers/data_refinery_workers/processors/qn_reference.py

```diff
@@ -210,6 +210,20 @@ def _create_result_objects(job_context: Dict) -> Dict:
     job_context['success'] = True
     return job_context
 
+def _update_experiment_caches(job_context: Dict) -> Dict:
+    """ Experiments have a cached value with the number of samples that have QN targets
+        generated, this value should be updated after generating new QN targets. """
+    unique_experiments = []
+    all_samples = job_context['samples']['ALL']
```

Comment:
These samples will be limited to microarray because we only build QN targets with microarray data. I think what you want to do is filter for experiments that have the organism that the QN target is for.